common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -157,6 +156,32 @@ public class RssBaseConf extends RssConf {
       .defaultValue(true)
       .withDescription("The switch for jvm metrics verbose");
 
+  public static final ConfigOption<Boolean> RSS_ACCESS_HADOOP_KERBEROS_ENABLE = ConfigOptions
+      .key("rss.access.hadoop.kerberos.enable")
+      .booleanType()
+      .defaultValue(false)
+      .withDescription("Whether enable visiting secured hadoop cluster.");
+
+  public static final ConfigOption<String> RSS_ACCESS_HADOOP_KERBEROS_KEYTAB_FILE = ConfigOptions
+      .key("rss.access.hadoop.kerberos.keytab.file")
+      .stringType()
+      .noDefaultValue()
+      .withDescription("The kerberos keytab file path. And only when "
+          + RSS_ACCESS_HADOOP_KERBEROS_ENABLE.key() + " enabled, the option will be valid.");
+
+  public static final ConfigOption<String> RSS_ACCESS_HADOOP_KERBEROS_PRINCIPAL = ConfigOptions
+      .key("rss.access.hadoop.kerberos.principal")
+      .stringType()
+      .noDefaultValue()
+      .withDescription("The kerberos keytab principal. And only when "
+          + RSS_ACCESS_HADOOP_KERBEROS_ENABLE.key() + " enabled, the option will be valid.");
+
+  public static final ConfigOption<Long> RSS_ACCESS_HADOOP_KERBEROS_RELOGIN_INTERVAL = ConfigOptions
```

Comment:
RSS_ACCESS_HADOOP_KERBEROS_RELOGIN_INTERVAL -> RSS_ACCESS_HADOOP_KERBEROS_RELOGIN_INTERVAL_SEC

Could we add unit to its name to help people understand the code easily?

Reply:
Got it.